dali/driver/atxled.py

```diff
@@ -0,0 +1,190 @@
+from dali.command import Command
+from dali.driver.base import SyncDALIDriver, DALIDriver
+from dali.frame import ForwardFrame, BackwardFrame
+import logging
+import sys
+
+# avoid import name collision with 'serial' driver
+original_sys_path = list(sys.path)
+sys.path = [p for p in sys.path if not p.endswith("python-dali/dali/driver")]
```

Comment:
This shouldn't be necessary. I suspect you're developing with `python-dali/dali/driver` as the CWD so you can run this module as `__main__`? Don't do that!

Reply:
Good point-- I removed this and removed __main__ since it was only there ad probationem